src/store/theme-context.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+
+export interface ThemeContextConfig {
+  theme: string;
+  toggleTheme?: () => void;
+}
+
+const themes: {
```

Comment:
as far as you are using typescript you could just create enum for theme, it both helps to add types (at least for useState in your case) can be used as regular variable.
also i'd recommend to replace empty string for `light` property to "light"

Reply:
Ok, changed to enum.